npm/private/npm_package_store_info.bzl

```diff
@@ -0,0 +1,15 @@
+"NpmPackageStoreInfo provider"
+
+NpmPackageStoreInfo = provider(
+    doc = "Provides an npm package store",
```

Comment:
this is just repeating the name, maybe say a bit more?

Reply:
✅ 
```
    doc = """Provides an npm package store
    
    This is a npm package within the virtual store of a pnpm-style symlinked node_modules tree.
    See https://pnpm.io/symlinked-node-modules-structure for more information about
    symlinked node_modules trees.""",
```